### PR TITLE
Russian language support- Ksenia Lukacher

### PR DESCRIPTION
**Pull request by alpita-masurkar**

### CODE REVIEW

**Comment by andreanuzzo** [comment]
Can you please check why it didn't build properly? Does it build on your end?

**Comment by alpita-masurkar** [comment]
> Can you please check why it didn't build properly? Does it build on your end?

Hi @andreanuzzo Yes it did. I have no idea why it didn't build here, so I also raised this with a couple of people. I think I'll just close this PR and create a new one and see how it goes.

File: app/locales/ru/notification.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "notification_main_text": "Проверяйте пересечения маршрутов ежедневно",
```
Comment (sergesemashko) [inline app/locales/ru/notification.json:2]: nit-picky: how about "Новые уведомления" for `"notification_main_text": "Recent notifications:"`? maybe I'm wrong but looks like this translation changes the meaning a bit from the original intention

File: app/locales/ru/notification.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "notification_title": "Анализируйте пересечения маршрутов",
```
Comment (sergesemashko) [inline app/locales/ru/notification.json:3]: nit-picky: how about "Пересечения маршрутов за 2 недели" for `2-Week Exposure Profile` translation?

File: app/locales/ru/locationTracking.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "start_logging":"НАЧАТЬ ЗАПИСЬ МЕСТОПОЛОЖЕНИЙ",
```
Comment (sergesemashko) [inline app/locales/ru/locationTracking.json:2]: it looks like `start_logging` was renamed to `home_start_tracking` with the V2 redesign. you might want to check out latest code from develop branch. Some other translations variables might be affected.

Comment (alpita-masurkar) [reply app/locales/ru/locationTracking.json:2]: ok will do... thank you

**Comment by alpita-masurkar** [comment]
@sergesemashko thank you for the review, I'll look into this and your suggestions and will make modifications

**Comment by kenpugsley** [comment]
Thank you for this translation.  The UI has changed significantly and has now finalized for the v1.0 release, which has resulted in a great deal of changes to the text in the final app.  Because of this we found that it was best to remove the existing translations and start again.  Even though we are also closing this PR, we hope you understand and will help us again.

The text in the app is now complete, so **we're asking for your help to redo this translation on the text for the v1.0 release**.  Our translation lead @alpita-masurkar will be coordinating this effort.

Thank you from the COVID Safe Paths dev team.

**Pull request closed without merging**